Source/Engine/McBopomofoLM.h

```diff
@@ -105,6 +105,12 @@ class McBopomofoLM : public Formosa::Gramambular2::LanguageModel {
     const std::vector<std::string> associatedPhrasesForKey(const std::string& key);
     bool hasAssociatedPhrasesForKey(const std::string& key);
 
+    /// Returns a list of pronounciations that match a given value.
+    /// @param value A string representing the text to look up pronounciation candidates for. For example,
+    ///     if you pass "說", it returns a list of records that include ㄕㄨㄛ, ㄕㄨㄟˋ, and ㄩㄝˋ.
+    /// @return An unsorted list of string records of the form "<key> <value> <score>" separated by spaces.
+    std::vector<std::string_view> getPronounciations(const std::string_view& value);
```

Comment:
Typo: pro_nun_ciation (even though the verb is to pro_noun_ce…), and this will need a global search/replace.

Alternatively, consider just calling it "reading", so `getReadings`, which is the term McBopomofo uses in other places.

Reply:
Heh, I was telling myself the word looks wrong as I was typing them... thanks!